test/rqrcode/export_ansi_test.rb

```diff
@@ -0,0 +1,12 @@
+require 'test_helper'
+require 'rqrcode/data'
+
+describe 'Export::ANSI' do
+  it 'must respond_to ansi' do
+    RQRCode::QRCode.new('x').must_respond_to :'as_ansi'
```

Comment:
Can this not just be `:as_ansi`? (likewise in other specs)

Reply:
Yes, well spotted. Copied from old project and not noticed